collector/loadavg.go

```diff
@@ -3,18 +3,17 @@
 package collector
 
 import (
+	"errors"
 	"fmt"
-	"io/ioutil"
-	"strconv"
-	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/log"
 )
 
-const (
-	procLoad = "/proc/loadavg"
-)
+// #include <stdlib.h>
+import "C"
+
+var loadavg [1]C.double
```

Comment:
Is it necessary to have this variable under the global scope? Could you move it into `getLoad1` instead?